### DISCUSSION TITLE
[Adonis 5] Validator & Error Reporter

### DISCUSSION THREAD

**Discussion by Melchyore** [adonisjs/core#1446] [Help]
Hi,

I'm trying to add a custom error reporter for my validator. So I created a new file in App/Validators/Reporters/DefaultReporter.ts and filled it with this content : [Creating your own error reporter](https://preview.adonisjs.com/guides/validator/error-reporters#creating-your-own-error-reporter) from the docs.

I get the following error : `Module '@ioc:Adonis/Core/Validator' has no exported member ValidationException`.

Also, is it possible to use the Error Reporter inside the Validator Class? Cuz If I use it inside `request.validate` method, I can't use the class version of the Validator for schema.

If I try to declare a public variable `public reporter = DefaultReporter` in the Validator Class, I get the following error : `TypeError: Validator_1.ValidationException is not a constructor at DefaultReporter.toError `

**EDIT**
This is how I fixed it : `import { ValidationException } from '@adonisjs/validator/build/src/ValidationException'` is it the best way?

**Comment by dare-samuel** [comment]
I am also having the same error... Please somebody help

**Comment by thetutlage** [comment]
Fixed in https://github.com/adonisjs/validator/commit/d7552f09307f632ca807dda65f2db68d0fe30a5b. But have to wait for the release
